src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -589,36 +587,33 @@ def _compute_offsets(self, token_ids, time_precision=0.02, longform_timestamps=N
             consecutive = np.append(consecutive, np.where(timestamp_tokens)[0][-1] + 1)
 
         last_slice = np.where(timestamp_tokens)[0][0]
-        for i, current_slice in enumerate(consecutive):
+        cur_max_timestamp = 0
```

Comment:
With no change to the overall processor/tokenizer design, we can fix the original timestamp issue by keeping track of the last timestamp predicted